src/index.js

```diff
@@ -8,30 +8,35 @@ const sass = require('gulp-sass');
 const watch = require('gulp-watch');
 const yargs = require('yargs').argv;
 
+const isObject = a => typeof a === 'object';
+
 class SassBuild {
     constructor(src, dest, gulp) {
         this._src = src;
         this._destinations =
             Array.isArray(dest)
                 ? dest
                 : [dest];
-                
+
         this._verbose = yargs.verbose;
         this._gulp = gulp;
 
         this.build = this.build.bind(this);
         this.watch = this.watch.bind(this);
     }
 
-    build() {
+    build(sassOptions, cssMinOptions) {
```

Comment:
This signature change would be a significant breaking change to the API and affect existing usage.

One of the guidelines I was using on this project and the other `gulp-*-build` projects is that the API must remain as simple and locked-down as possible so as to require the least amount of configuration when spinning up a project.

I've been contemplating adding hooks into specific sets of options for sass, cssmin, and autoprefixer, but I haven't had the time to explore their individual APIs to determine what level of configuration is appropriate for each.

More importantly with this change, the appropriate place to pass the options is via the constructor, rather than the build command.